src/util/customTransformer.js

```diff
@@ -253,8 +253,10 @@ async function runUserTransform(
     const tags = {
       identifier: 'v0',
       errored: transformationError ? true : false,
-      ...(events.length && events[0].metadata ? getMetadata(events[0].metadata) : {}),
-      ...(events.length && events[0].metadata ? getTransformationMetadata(events[0].metadata) : {}),
+      ...(Object.keys(eventsMetadata).length ? getMetadata(Object.values(eventsMetadata)[0]) : {}),
+      ...(Object.keys(eventsMetadata).length
+        ? getTransformationMetadata(Object.values(eventsMetadata)[0])
+        : {}),
```

Comment:
Can you explain the issue here a bit ?

Reply:
v0 execute function receives only event messages. metadata is not attached to them. Earlier tag extraction `getMetadata(events[0].metdata)` always returns nil. So changed extraction to eventsMetadata